lib/collection.ts

```diff
@@ -9,3 +9,52 @@ export class Collection {
 		return this.list[Math.floor(Math.random() * this.list.length)];
 	}
 }
+
+const materials = {
+	"A dripping liquid oil painting": ["realistic", "modern"],
```

Comment:
This creates an error

```ts
import { materials, styles } from "../lib/collection";

const material__dripping_liquid_oil_painting =
	materials["dripping liquid oil painting"];
console.log(material__dripping_liquid_oil_painting);

const style_from_material__modern = material__dripping_liquid_oil_painting[1];
console.log(style_from_material__modern);

const color_from_style__modern = styles[style_from_material__modern];
console.log(color_from_style__modern);

```

Reply:
@julizet  I think that's because the additionally added "a" or "an" in the material keys

Reply:
it was indeed missing. I also found that "conceptual art" was missing and added both within the latest commit